20_1 button: add getting started (#1773)
DevExpress/devextreme-documentation: concepts/05 UI Components/Button/00 Getting Started with Button/04 Handle the Click Event.md

> To respond to user clicks, write an [onClick](/Documentation/ApiReference/UI_Components/dxButton/Configuration/#onClick) handler. This tutorial displays a toast message:

Reviewer: "This tutorial displays a toast message" How can a tutorial display a message?

> You can implement any behavior that occurs when a user clicks a Button. For this, use the [onClick](/Documentation/ApiReference/UI_Components/dxButton/Configuration/#onClick) handler. In this tutorial, we will display a toast message:

Reviewer 2: Seems a bit too much text to describe clicks, so I tried to shorten it. I also removed the word "we" - I explain why in another comment below. 

```suggestion
To respond to user clicks, write an [onClick](/Documentation/ApiReference/UI_Components/dxButton/Configuration/#onClick) handler. This tutorial displays a toast message:
```
